webrtc.html

```diff
@@ -3017,14 +3017,14 @@
         <h4>RTCSessionDescriptionCallback</h4>
         <div>
           <pre class="idl">
-          callback RTCSessionDescriptionCallback = void (RTCSessionDescription sdp);</pre>
+          callback RTCSessionDescriptionCallback = void (RTCSessionDescriptionInit description);</pre>
           <section>
             <h2>Callback <a class="idlType">RTCSessionDescriptionCallback</a>
             Parameters</h2>
             <dl data-link-for="RTCSessionDescriptionCallback" data-dfn-for=
             "RTCSessionDescriptionCallback" class="callback-members">
-              <dt><code>sdp</code> of type <span class=
-              "idlMemberType"><a>RTCSessionDescription</a></span></dt>
+              <dt><code>description</code> of type <span class=
+              "idlMemberType"><a>RTCSessionDescriptionInit</a></span></dt>
```

Comment:
It can certainly be observed, by `desc instanceof RTCSessionDescription` or trying to set `type` or `sdp` to something other than a string. The question is whether it's likely to break stuff. Unclear.

There seems to be a bunch of related changes here, and IMHO someone needs to have a look at them as a whole and see if they're likely to work out or not, adding use counters. What I did in https://codereview.chromium.org/2490543002 doesn't cover it all I think.